README: Add Communication section with Forum information (#119)
ansible-collections/community.hrobot: README.md

Please note that this collection does **not** support Windows targets.

## Communication

* Join the Ansible forum:
  * [Get Help](https://forum.ansible.com/c/help/6): get help or help others.

>   * [Posts tagged with 'your tag'](https://forum.ansible.com/tag/YOUR_TAG): subscribe to participate in collection-related conversations.
>   * [Refer to your forum group here if exists](https://forum.ansible.com/g/): by joining the team you will automatically get subscribed to the posts tagged with [your group forum tag here](https://forum.ansible.com/tags).

Reviewer: ```suggestion
  * [Posts tagged with 'hrobot'](https://forum.ansible.com/tag/hrobot): subscribe to participate in Hetzner Robot related conversations.
```
